content/nginx-one/secure-your-fleet/secure.md

```diff
@@ -0,0 +1,131 @@
+
+## Configure an active alert policy
+
+The NGINX One Console monitors all connected NGINX instances for CVEs and insecure configurations. Using the F5 Distributed Cloud's Alert Policies you can receive alerts for these risks in a manner of your choosing; for the purposes of this guide we will show you how to configure E-Mail alerts.
+
+The F5 Distributed Cloud generates alerts from all its services including NGINX One. You can configure rules to send those alerts to a receiver of your choice. These instructions walk you through how to configure an email notification when we see new CVEs or detect security issues with your NGINX instances.
+
+This page describes basic steps to set up an email alert. For authoritative documentation, see
+[Alerts - Email & SMS](https://docs.cloud.f5.com/docs-v2/shared-configuration/how-tos/alerting/alerts-email-sms).
+
+## Configure alerts to be sent to your email
+
+To configure security-related alerts, follow these steps:
+
+1. Navigate to the F5 Distributed Cloud Console at https://INSERT_YOUR_TENANT_NAME.console.ves.volterra.io. 
+1. Select **Audit Logs & Alerts**
+1. Select **Manage > Alert Receiver**
+1. Configure the **Alert Receivers**
+   1. Enter the name of your choice.
+   1. (Optional) Specify a label and description.
+1. Under **Receiver**, select Email and enter your email address.
+1. Select **Add Alert Receiver**
+1. Your Email receiver should now appear on the list of Alert Receivers.
+1. Under the Actions column, select Verify Email.
+1. Select **Send email** to confirm.
+1. You should receive a verification code in the email provided. Copy that code.
+1. Under the Actions column, select **Enter verification code**.
+1. Paste the code and select **Verify receiver**.
+
+## Configure Alert Policy
+
+Next, configure the policy that identifies when you'll get an alert. 
+
+1. Navigate to **Alerts Management > Alert Policies**.
+1. Select **Add Alert Policy**.
+   1. Enter the name of your choice.
+   1. (Optional) Specify a label and description.
+1. Under Alert Reciever Configuration > Alert Receivers, select the Alert Receiver you just created.
+1. Under Policy Rules select Configure.
+1. Select Add Item.
+1. Under Select Alerts (TBD).
+1. Set the Action as Send and select Apply.
+
+Now set a second alert related to Common Vulnerabilities and Exposures (CVEs).
+
+1. Select Add Item
+1. Under Select Alerts {adding additional Alert type for CVE)
+1. Set the Action as Send and select Apply
+1. Select **Save and Exit**
+
+You've now set up F5 Distributed Cloud to send you security-related alerts from NGINX One Console.
+
+## Summary
+
+In this tutorial, you learned how to:
+
+- Access the NGINX One Console
+- Connect an NGINX instance
+- Configure an Alert
+
+You will now receive an email any time the NGINX One Console recognizes one of your connected instances has a configuration that is insecure or an NGINX CVE impacts it.
```

Comment:
Style guide says to use "affect" instead of "impact" for verbs: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/i/impact

Not the biggest deal but "any time the NGINX One Console recognizes ..." is anthropomorphic, which the style guide recommends against.

```suggestion
You will now receive an email any time the NGINX One Console recognizes one of your connected instances has a configuration that is insecure or an NGINX CVE affects it.
```